README.md

```diff
@@ -1,2 +1,62 @@
 # Hello_Microverse
 Hello World
+
+
+
+
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo 
+
+[Coming Soon]
+
+
+## Getting Started
+
+**Go to your Cli and follow the steps below**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone the repository: go to Your Cli and run "git clone <repository url>"
+
+- change into the repository:  run 'cd <repository name>'
+
+- Install all the project dependences: Run 'npm Insttall' in your cli
+
+- now you can view the project in the browser using any web server of choice
+
+
+
+## Author
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/Eugeneiyukhun)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/eugene-iyukhun-bb491612a)
+- Twitter: [twitter](https://twitter.com/EIyukhun)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to microverse and anyone whose code was used
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] Your README file is professional 💯 However, I noticed that the MIT/LICENSE link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯